src/ui/utils/formatters.ts

```diff
@@ -25,6 +25,44 @@ const formatTimeToSec = (date: string) => {
   return new Date(date).toLocaleTimeString("en-GB");
 };
 
+const timeDifference = (timestamp: string) => {
+  const startDate = new Date(timestamp);
+  const endDate = new Date(); // Current date and time in local time zone
+  const startDateUTC = Date.UTC(
```

Comment:
I had to do that conversion otherwise there will be a misalignment between tests in my local and tests on GitHub (there will be a time difference). See these tests failed here:

https://github.com/cardano-foundation/cf-identity-wallet/actions/runs/9696996519/job/26760130153

They were all good in my local but failing on GitHub.


That is already all covered in the tests here:

![28062024090832](https://github.com/cardano-foundation/cf-identity-wallet/assets/119612231/bbb4905f-4908-4bba-9178-4422fc98402b)

See it's all green:

![28062024090719](https://github.com/cardano-foundation/cf-identity-wallet/assets/119612231/6e551c11-df3e-40e2-9d8c-b690d0152b7d)


I can add some dedicated tests just for that if this is what you want, but they will be duplicates.

Reply:
Just added that to formatters tests 👍

Reply:
thanks for the clarification!